security/pkg/nodeagent/caclient/credentials.go

```diff
@@ -16,20 +16,28 @@ package caclient
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"strings"
 
 	"google.golang.org/grpc/credentials"
 
 	"istio.io/istio/pkg/security"
+	"istio.io/istio/security/pkg/stsservice"
+	"istio.io/istio/security/pkg/stsservice/server"
 	"istio.io/pkg/log"
 )
 
+const (
+	scope = "https://www.googleapis.com/auth/cloud-platform"
+)
```

Comment:
Move this constant to Google plugin.

Reply:
Revised to use the constant in Google plugin.